UI/MainMenu/InGameMenu.gd

```diff
@@ -1,45 +1,22 @@
-extends Control
+extends TabContainer
 
-onready var loadGameScreen = $LoadGameScreen
-onready var mainMenuScreen = $MainMenu
-onready var saveMenuScreen = $SaveGameScreen
 signal onResumeButtonPressed
 
-# Called when the node enters the scene tree for the first time.
-func _ready():
-	pass # Replace with function body.
-
-
-# Called every frame. 'delta' is the elapsed time since the previous frame.
-#func _process(delta):
-#	pass
-func hideAllMenus():
-	mainMenuScreen.visible = false
-	loadGameScreen.visible = false
-	saveMenuScreen.visible = false
-
 func _on_LoadGameButton_pressed():
-	hideAllMenus()
-	loadGameScreen.visible = true
-
-func _on_LoadGameScreen_onClosePressed():
-	hideAllMenus()
-	mainMenuScreen.visible = true
-
+	current_tab = 1
 
 func _on_MainMenuButton_pressed():
-	var _ok = get_tree().change_scene("res://UI/MainMenu/MainMenu.tscn")
-
+	GM.main.runScene("MainMenu")
+	GM.main.runCurrentScene()
```

Comment:
You didn't clear the scene stack and then add a main menu scene on top which is bad

Reply:
Because I don't know how it works! In code it's bit unclear. (About modding wiki I know only at the end of developing UI)